config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root to: 'pages#home'
+  get 'entries', to: 'pages#entries', as: 'entries'
+  get 'tasks', to: 'pages#tasks', as: 'tasks'
+  get 'goals', to: 'pages#goals', as: 'goals'
+  get 'profile', to: 'pages#profile', as: 'profile'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
-end
+  get 'new_entry_form', to: 'pages#new_entry_form', as: 'new_entry_form'
+  post 'create_entry', to: 'pages#create_entry', as: 'create_entry'
+end
```

Comment:
Routes really complicated. Try "resources :pages"

Reply:
please add an empty line at the end of the file

Reply:
I agree with Melvin, try resources :pages this should give you all the built in methods